lib/encrypt_attributes.rb

```diff
@@ -39,6 +39,21 @@ def encrypted_attribute(name, options={})
         value = send(name)
         value.respond_to?(:empty?) ? !value.empty? : !!value
       end
+
+      # Define methods using serialized accessors option
+      if options[:serialize].is_a?(Hash) && options[:serialize].has_key?(:accessors) && options[:serialize][:accessors].is_a?(Array)
+        options[:serialize][:accessors].map(&:to_sym).each do |accessor|
+          accessor_name = "#{name}_#{accessor}"
+
+          define_method(accessor_name) do
+            send(name)[accessor]
+          end
+
+          define_method("#{accessor_name}=") do |val|
+            send("#{name}=", { "#{accessor}": val })
```

Comment:
Sorry my suggestion is backward, you need to merge in `{ accessor.to_sym => val }` to `send(name) || {}`:

```ruby
send("#{name}=", (send(name) || {}).merge!({ accessor.to_sym => val })
```

We should have specs checking that this is working correctly.